examples/lib/cache.js

```diff
@@ -1,7 +1,39 @@
 var setupCache = function() {
+  let newWorker; // When sw.js is changed, this is the new service worker generated.
+
+  // Toggle a CSS class to display a popup prompting the user to fetch a new version.
+  function showUpdateModal() {
+    $('#update-prompt-modal').addClass('show');
+  }
+
+  // When a new service worker has been loaded, the button in the update prompt
```

Comment:
Please use multi-line comments instead of multiple inline comments. That will make it easier to read. Also, please keep words like service and worker on the same line.

Reply:
Got it, thanks!